subiquity/common/dmidecode.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2023 Canonical, Ltd.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+from subiquitycore.utils import run_command
+
+
+def dmidecode_get(key: str) -> str:
+    """Retrieve a value from `dmidecode -s`
+    :param key: dmidecode(8) KEYWORD
+    :returns: stripped string output, or empty string on failure
+    """
+    sp = run_command(["dmidecode", "-s", key], check=False, text=True)
```

Comment:
should we add dmidecode to the apt dependencies list ? 

If the command is not found, this will raise a `FileNotFoundError`.